server/dataone_upload.py

```diff
@@ -99,7 +106,8 @@ def create_upload_eml(tale, client, user, item_ids, license_id, file_sizes=dict(
                                  item_ids,
                                  eml_pid,
                                  file_sizes,
-                                 license_id)
+                                 license_id,
+                                 user_id,)
```

Comment:
is Python cool with this slash after `user_id`?

Reply:
Weird that totally worked. I don't really like it though. Fixed in 
abe2661b9fa45fc22c6fcbad982c6d81826c2dbc